pkg/utils/log/log.go

```diff
@@ -68,30 +67,30 @@ func setLogger(v *int, debug *bool) {
 		encoderConf := zap.NewDevelopmentEncoderConfig()
 		encoderConf.EncodeLevel = zapcore.CapitalColorLevelEncoder
```

Comment:
Anyone knows if this is necessary? `controller-runtime` already defaults to this:
```go
encCfg := zap.NewDevelopmentEncoderConfig()
o.Encoder = zapcore.NewConsoleEncoder(encCfg)
```
So if this is not needed, we can remove the whole `if dev.Enabled` clause

Reply:
We add the colour level encoder in debug -- which doesn't happen in the controller-runtime configuration. It's nice to have :)

Reply:
ok :)